prlworkflows/tests/test_sqs.py

```diff
@@ -247,21 +247,21 @@ def test_sqs_is_properly_enumerated_for_a_higher_order_sublattice_model():
     """Tests that a sublattice model of higher order than an SQS properly enumerated"""
     structure = lat_in_to_sqs(ATAT_FCC_L12_LATTICE_IN)
     structures = enumerate_sqs(structure, [['Al', 'Ni'], ['Fe', 'Cr']], endmembers=False)
-    assert len(structures) == 4
+    assert len(structures) == 2
```

Comment:
I think we also said that these should create the proper endmembers, right? So this should generate

```python
[['Al'], ['Fe']]
[['Al'], ['Cr']]
[['Ni'], ['Fe']]
[['Ni'], ['Cr']]
[['Al', 'Ni'], ['Fe']]
[['Al', 'Ni'], ['Cr']]
```

given the symmetry of the FCC_L12 structure. Is that right?

You should also make a new test that tests a non-symmetric case that should generate 

```python
[['Al'], ['Fe']]
[['Al'], ['Cr']]
[['Ni'], ['Fe']]
[['Ni'], ['Cr']]
[['Al', 'Ni'], ['Fe']]
[['Al', 'Ni'], ['Fe']] # with different occupancies
[['Al', 'Ni'], ['Cr']]
[['Al', 'Ni'], ['Cr']] # again with different occupancies 
```